### ISSUE TITLE
Make Blueprint package optional

### ISSUE THREAD

**Issue by driesvints** [dingo/api#1323]
Is there any chance we can make the blueprint package not a required dependency? It seems to be overkill to require it when you're not using it. Also further separates dependencies which is always a good thing.

**Comment by alexcroox** [comment]
Agreed, I'm using Swagger so it's a shame to have Blueprint required
